app/components/field-validations-form.js

```diff
@@ -3,7 +3,8 @@ import { tracked } from '@glimmer/tracking';
 import { restartableTask, timeout } from 'ember-concurrency';
 import { getFieldAndValidationTriples } from '../utils/get-field-and-validation-triples';
 import { getValidationSubjectsOnNode } from '../utils/forking-store-helpers';
-import { EXT, FORM } from '../utils/rdflib';
+import { EXT } from '../utils/namespaces';
```

Comment:
Renamed our _rdflib.js_ file to _namespaces.js_. This is how it is called in the `ember-submission-form-field-helpers`. Our namespaces are extending the ones defined in the helper addon.